src/stylesheets/components/BlogSection/BlogPost.sass

```diff
@@ -1,71 +1,63 @@
 .post
-  border-radius: 0
-  border: none
-  cursor: pointer
-  margin-top: $marg-4
-  width: 100%
+    cursor: pointer
+    margin-top: $marg-4
 
-  &__img-container
-    background-color: $color-neutral-300
-    height: auto
-    overflow: hidden
+    &__img
+        height: auto
+        width: 100%
 
-  &__img
-    height: auto
-    width: 100%
-
-  &__content
-    background-color: $color-neutral-300
-    color: $color-neutral-700
-    padding: $marg-3 $marg-3 $marg-4
-    font-size: $font-size-xl
+    &__content
+        background-color: $color-neutral-300
+        color: $color-neutral-700
+        font-size: $font-size-xl
+        margin-top: -$marg-base
+        padding: $marg-3 $marg-3 $marg-4
 
-  &__title
-    font-size: $font-size-2xl
-    font-family: Raleway
-    display: -webkit-box
-    -webkit-line-clamp: 2
-    -webkit-box-orient: vertical
-    overflow: hidden
+    &__title
+        font-size: $font-size-2xl
+        font-family: Raleway
+        display: -webkit-box
+        -webkit-line-clamp: 2
+        -webkit-box-orient: vertical
+        overflow: hidden
 
-  &__info
-    @include flex($main: flex-start)
-    font-size: $font-size-base
-    margin: $marg-base 0
+    &__info
+        font-size: $font-size-base
+        margin: $marg-base 0
 
-  &__author
-    margin-right: 5px
+    &__author
+        margin-right: $marg-1
 
-  &__text
-    font-size: $font-size-sm
-    margin-bottom: -20px
+    &__text
+        font-size: $font-size-lg
+        margin-bottom: $marg-3  
 
 @media only screen and (max-width: $xxl)
-  .post
-    &__title
-      font-size: $font-size-xl  
+    .post
+        &__title
+            font-size: $font-size-xl  
 
-    &__content
-      font-size: $font-size-lg 
+        &__content
+            font-size: $font-size-lg 
 
-    &__info
-      font-size: $font-size-sm
+        &__info
+            font-size: $font-size-sm
 
 
 @media only screen and (max-width: $l)
-  .post
-    width: 100%
+    .post
+        width: 100%
```

Comment:
This `width: 100%` could be useful on all resolutions - if there is no text in `post__text` and the title is short, the post will not take full width of the screen:
![Screenshot 2023-03-19 at 13 05 39](https://user-images.githubusercontent.com/2543423/226174040-31540a18-da4a-4098-9428-ee971ac25798.png)
But please remove it, there is a better fix or it - see `.blog-grid`.